team/quiet.rst

```diff
@@ -0,0 +1,40 @@
+
+############
+Quiet Friday
+############
+
+Starting on 2020-09-25 DMLT will commence **Quiet Friday**. 
```

Comment:
I suggest not hard-coding the date in the Developer Guide — it won't be relevant in 8 days. Let's just put it in the announcement.

Reply:
Historic moment!
But ok